addons/web/static/src/less/kanban_dashboard.less

```diff
@@ -1,138 +1,264 @@
+        padding: 3px @link-padding-gap;
+        color: @text-color;
+        display: block;
     }
 
-    &.o_kanban_dashboard.o_kanban_ungrouped, &.o_kanban_dashboard.o_kanban_grouped {
+    &:hover > a {
+        background-color: @gray-lighter-dark;
+        color: @headings-color;
+    }
 
-        // If invisible, elem will no take more than its original height
-        a, span {
-            .o-text-overflow(inline-block);
-            position: relative;
-        }
+    &:last-child {
+        margin-bottom: 5px;
+    }
+}
 
-        a {
-            &.o_inline_link {
-                display: inline;
-            }
-            &:focus, &:active, &:focus:active {
-                outline: none;
-            }
-        }
 
+.o_kanban_view {
+    &.o_kanban_dashboard {
         .o_kanban_record {
-            padding: 0;
+            padding: @o-kanban-dashboard-vpadding @o-kanban-dashboard-hpadding;
             position: relative;
+            .o-flex-display();
+            .o-flex-flow(column, nowrap);
+            .o-justify-content(space-between);
+
             @media (max-width: @screen-xs-max) {
                 margin-bottom: 10px;
             }
 
-            .row .row {
-                margin-left: 0px;
-                margin-right: 0px;
+            // ------- Generic layout adaptations -------
+            .container {
+                max-width: 100%;
+            }
 
-                > div {
-                    padding-left: 0px;
-                    padding-right: 0px;
-                }
+            // ------- Dropdown toggle & menu -------
+            .o_kanban_manage_toggle_button {
+                .o-kanban-dropdown(@o-kanban-dashboard-hpadding);
             }
-        }
 
-        .o_kanban_card_header {
-            .o-flex-display();
-            position: relative;
+            .o_kanban_card_manage_pane {
+                .dropdown-menu;
+                .o-kanban-dropdown-menu();
 
-            .o_kanban_card_header_title {
-                .o-flex(3, 0, 0);
-                padding: @odoo-horizontal-padding/2 @odoo-horizontal-padding;
+                // Arbitrary value to place the dropdown-menu
+                // exactly below the dropdown-toggle
+                top: 34px;
 
-                .o_primary {
-                    font-weight: 700;
+                > div:not(.o_project_kanban_colorpicker) {
+                    padding: 3px 0 3px 20px;
+                    visibility: visible;
+                    margin-bottom: 5px;
                 }
+
+                > .o_kanban_card_manage_section {
+                    border-bottom: 1px solid @gray-lighter-dark;
+                    margin-bottom: 10px;
+
+                    > div {
+                        .o-kanban-dashboard-dropdown-link();
+                    }
+                }
+
+                // Dropdown menu with complex layout
+                &.container {
+                    max-width: 400px;
```

Comment:
to avoid that the `.container` class take the bootstrap-size and media queries.
We should remove that class but there are all the inner row, cols...
Also I'm worried about xpath issues :/  
There is another `max-width: 100%` before, but the dropdown will be too large in some resolutions...